requirements.txt

```diff
@@ -1,3 +1,7 @@
 # Note: Add any additional requirements to setup.py's install_requires field
 --editable .
+json==3.9.4
+
+pandas==1.2.4
+python-pptx==0.6.18
```

Comment:
Please read the comment at the top of this file.

Reply:
Also, why did you move all the source out of `src` into `cyhy/pe-reports`?  That is how it is done in [cisagov/cyhy-mailer](https://github.com/cisagov/cyhy-mailer), but that code pre-dates the skeletons and has not yet been brought into the light.

Reply:
Your guess is correct, it is exactly why I changed src. Due to other cyhy repos. 
I was unaware the cyhy repos are older and not necessarily taken form "skeletons". We want to stay compliant so I will change to src/pe-reports. Also to give scope, we are working through some processes, a couple of steps beyond POC and coordinating some source control. It'll be a little rough in the interim, but plan to keep up with standards as we go.